data/adityassharma-ss.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Its best to separate two different additions into two different prs could you remove the profile addition from this pr and include it into another one?

Reply:
## Smaller and focused Pull Requests (PR)

### TLDR;

The smaller and more focused the PR the easier for the reviewer and the more likely it is to to merge faster.

---

### Why are smaller PRs better?

- Easier for the reviewer
- Will get merged faster
- Reduces conflicts with other changes
- You get more green squares if you separate PRs out into focused changes

---
**Summary:** :hammer_and_wrench: This code needs a fix